### PR TITLE
Status: Display indicators of edited posts 

### PR DESCRIPTION
**Pull request by fruyek**
Screenshot_20221125-181626

### CODE REVIEW

**Comment by fruyek** [comment]
> Could you please try to also add it to the accessibility description? at least in the full view (see `setDescriptionForStatus()`)

I think I already did that in https://github.com/tuskyapp/Tusky/pull/2935/commits/fcfc058ace6be7736a711076b53a53d2df8a998f. Do I have to add it somewhere else as well?

**Comment by charlag** [comment]
No, it's perfect, I was tired and missed that, sorry!

File: app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java [hunk 1]
```diff
@@ -318,19 +318,24 @@ private void setAvatar(String url,
...
             } else {
                 long then = createdAt.getTime();
                 long now = System.currentTimeMillis();
                 String readout = TimestampUtils.getRelativeTimeSpanString(timestampInfo.getContext(), then, now);
-                timestampInfo.setText(readout);
+                timestampText = readout;
             }
         }
+
+        if (editedAt != null)
```
Comment (Gryzor) [inline app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java:336]: I'd say if you don't mind use the brackets {} as the project uses them everywhere, even for one line if statements (at least that has been my limited experience). 

There's more instances of this, so I will not spam you with comments, but I'm sure the project maintainer will care for the consistency.

**Review by connyduck — CHANGES_REQUESTED** [review]
Can you please resolve the merge conflicts so I can merge it? @fruyek

**Review by connyduck — APPROVED** [review]

**Comment by fruyek** [comment]
@connyduck done

**Pull request merged**